en/installation_guide/install_in_symfony_project.rst

```diff
@@ -0,0 +1,580 @@
+
+    fos_user:
+        resource: "@FOSUserBundle/Resources/config/routing/all.xml"
+
+    root:
+        path: /
+        methods: ['GET']
+        defaults:
+            _controller: FrameworkBundle:Redirect:urlRedirect
+            path: /admin
+            permanent: true
+
+    open_orchestra_base:
+        resource: "@OpenOrchestraBaseBundle/Resources/config/routing.yml"
+
+Configure the security
+
+.. code-block:: yaml
+
+    # security.yaml
+    encoders:
+        FOS\UserBundle\Model\UserInterface: sha512
+
+    role_hierarchy:
+        ROLE_ADMIN:       ROLE_USER
+        ROLE_SUPER_ADMIN: [ROLE_USER, ROLE_ADMIN, ROLE_ALLOWED_TO_SWITCH]
+
+    providers:
+        fos_userbundle:
+            id: fos_user.user_provider.username
+
+    access_decision_manager:
+        strategy: unanimous
+
+    firewalls:
+        dev:
+            pattern:  ^/(_(profiler|wdt)|css|images|js)/
+            security: false
+        api:
+            pattern: ^/api/
+            oauth2: ~
+            anonymous: false
+            security: true
+            context: openorchestra
+        main:
+            pattern: ^/
+            form_login:
+                provider: fos_userbundle
+                csrf_provider: security.csrf.token_manager
+            anonymous: true
+            context: openorchestra
+            logout:
+                path:   /logout
+                target: /admin
+
+    access_control:
+        - { path: ^/login$, role: IS_AUTHENTICATED_ANONYMOUSLY }
+        - { path: ^/admin/, role: ROLE_USER }
+        - { path: ^/api/, role: ROLE_USER }
+
+
+Manage assets
+~~~~~~~~~~~~~
+
+Configure grunt to generate all assets used by Open Orchestra
+
+First, download grunt task folder of repository `open-orchestra/open-orchestra <https://github.com/open-orchestra/open-orchestra/tree/master/grunt>`_
+
+You can use subversion to export grunt folder in your folder application:
+
+.. code-block:: bash
+
+    svn export https://github.com/open-orchestra/open-orchestra/trunk/grunt
+
+After, Create file ``Gruntfile.js`` in the root folder of your application
+
+.. code-block:: javascript
+
+    module.exports = function(grunt) {
+      var appConfig = require('./grunt/app_config.js');
+      var GruntConfigBuilder = require(appConfig.GruntConfigBuilder);
+
+      GruntConfigBuilder.init(grunt, appConfig);
+    };
+
+Finally, run the grunt command to generate all assets
+
+.. code-block:: bash
+
+    ./bin/grunt
+
+Load the fixtures
+-----------------
+
+Open Orchestra needs some fixtures to work (an admin user, a website, ...).
+
+.. code-block:: bash
+
+    $ php app/console orchestra:mongodb:fixtures:load --type=production --env=prod
+
+
+Front
+-----
+
+Install the Bundle via Composer
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Add bundles used by the back office in your ``composer.json``
+
+.. code-block:: javascript
+
+    "require": {
+        // others dependencies
+        "open-orchestra/open-orchestra-front-bundle": "1.2.*@dev",
```

Comment:
this part deal with 1.2@dev, whereas the back part deal with 1.1. Maybe some adjustment here to be coherent